src/components/swap/SwapButtons/SwapButtons.tsx

```diff
@@ -176,3 +305,11 @@ export function SwapButtons({
     ></SwapButton>
   )
 }
+
+/**
+ * @param trade - the whole trade class, whatever it may contain
+ * @returns boolean - if the input currency exists and is native
+ */
+function isCurrencyNative(currency: Currency, chainId: ChainId) {
+  return Currency.getNative(chainId) == currency ? true : false
+}
```

Comment:
Don't we already have a `isNative` method from `Currency`? It seems we don't need this 😄

Reply:
Removed.